### PR TITLE
Cover/cmds/core/shutdown: Add tests.

### PR DESCRIPTION
**Pull request by llogen**
Added tests for shutdown.go in shutdown_test.go.

Signed-off-by: llogen <christophlange95@googlemail.com>

### CODE REVIEW

**Comment by llogen** [comment]
For this package i wanted to use the systemcall filter pkg. Due to the fact, that this pkg status is on hold, i couldn't continue working on this.

File: cmds/core/shutdown/shutdown_linux.go
Comment (rminnich) [inline cmds/core/shutdown/shutdown_linux.go:91]: you might change this:
shutdown(os.Args[1:]...)
see above.

File: cmds/core/shutdown/shutdown_linux.go
Comment (rminnich) [inline cmds/core/shutdown/shutdown_linux.go:50]: I think this would be more readable if you were not passing the program name in :-), and since you allow it to have 0 args: 
func shutdown(args ...string) (uint, error) {

File: cmds/core/shutdown/shutdown_linux.go
Comment (rminnich) [inline cmds/core/shutdown/shutdown_linux.go:51]: this len(args) == 1 is then a bit less confusing, I believe:
if len(args) == 0 {
args = []string{"halt"}
}

File: cmds/core/shutdown/shutdown_linux.go
Comment (rminnich) [inline cmds/core/shutdown/shutdown_linux.go:54]: from here on down, adjust indexes to [0], etc. etc.

**Review by rminnich — DISMISSED** [review]
see my comments, I would prefer the changes I advise here, but I leave it to you.

File: cmds/core/shutdown/shutdown_linux.go
Comment (ChriMarMe) [reply cmds/core/shutdown/shutdown_linux.go:91]: Cant mix os.Args[1:] and ...string

**Review by jensdrenhaus — APPROVED** [review]

**Pull request merged**
